### ISSUE TITLE
issues in execution of the project

### ISSUE THREAD

**Issue by saisree300** [hacky1997/voice-based-email-for-blind#14]
![project](https://user-images.githubusercontent.com/70372339/91547360-6b443680-e941-11ea-9dcf-f7187d9c3db9.png)

**Comment by saisree300** [comment]
Help me how to solve this error

**Comment by ankitsharma21598** [comment]
> Help me how to solve this error

is that audio speaks multiple times with incomplete 'text' sentences???

**Comment by hacky1997** [comment]
> > Help me how to solve this error
> 
> is that audio speaks multiple times with incomplete 'text' sentences???

No. And one thing don't comment on everyone's issue. Create your own issue and response on there.

**Comment by hacky1997** [comment]
> Help me how to solve this error

You have put it in the wrong directory and if it requires administration level then try with it and one more thing as this is open source project and it works fine on me, so before gets any exception to try to find out the solution from yourself read my documentation nicely then if you find something unrealistic or bug in my code then message me. We are all busy with our works! ;)

**Comment by chitti-anu** [comment]
@ankitsharma21598
 sir u have cleared that issue so will u provide me the solution for that
which package u r used and tell me how will u resolve that issue.i have
shot time for submitting sir.
